### PR TITLE
Fix a few dependabot warnings

### PR DESCRIPTION
**Pull request by FrederikBolding**
Fixes a few dependabot warnings by bumping `vite` and `find-babel-config`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
     "@lavamoat/lavapack@^3.3.0": "patch:@lavamoat/lavapack@npm%3A3.3.0#./.yarn/patches/@lavamoat-lavapack-npm-3.3.0-5a61b5374d.patch",
     "@types/mocha@^10.0.1": "patch:@types/mocha@npm:10.0.1#.yarn/patches/@types-mocha-npm-10.0.1-7c94e9e170.patch",
     "clet@^1.0.1": "patch:clet@npm%3A1.0.1#./.yarn/patches/clet-npm-1.0.1-8523231bdc.patch",
+    "find-babel-config": "^2.0.0",
```
Comment (Mrtenz) [inline package.json:49]: Can we upstream this?

Comment (FrederikBolding) [reply package.json:49]: I can try!

Comment (FrederikBolding) [reply package.json:49]: I have opened an upstream PR

Comment (FrederikBolding) [reply package.json:56]: We should merge this now though, we can remove the resolution if my upstream PR is merged.

**Review by david0xd — APPROVED** [review]

**Pull request merged**